internals/demo/init.go

```diff
@@ -33,13 +32,14 @@ func NewInitCommand(io ui.IO, newClient newClientFunc) *InitCommand {
 }
 
 // Register registers the command, arguments and flags on the provided Registerer.
-func (cmd *InitCommand) Register(r command.Registerer) {
+func (cmd *InitCommand) Register(r cli.Registerer) {
 	clause := r.Command("init", "Create the secrets necessary to connect with the demo application.")
 	clause.HelpLong("demo init creates a repository with the username and password needed to connect to the demo API.")
 
-	clause.Flag("repo", "The path of the repository to create. Defaults to a "+defaultDemoRepo+" repo in your personal namespace.").SetValue(&cmd.repo)
+	clause.Flags().VarPF(&cmd.repo, "repo", "", "The path of the repository to create. Defaults to a "+defaultDemoRepo+" repo in your personal namespace.")
 
-	command.BindAction(clause, cmd.Run)
+	clause.BindAction(cmd.Run)
+	clause.BindArguments(nil)
```

Comment:
Why is this line necessary?

Reply:
I think it was left from previous version of binding. It is no longer needed, so I removed it.